protos/utils.proto

```diff
@@ -50,10 +50,10 @@ message Prefixed {
 // such as Verilog and SPICE.
 message ParamValue {
   oneof value {
-    bool boolean = 1;           // Boolean
-    int64 integer = 2;          // Int
-    double floating_point = 3;  // Float
-    string string = 4;          // String-valued Param
+    bool boolean = 1;
+    int64 integer = 2;
+    double floating_point = 3;
```

Comment:
Should this be, like, `double_value` instead?  
I believe `float` and `double` are both protobuf types. 

Seems we may want a similar `_val` or `_value` or some-such suffix on some of these other variants too.

Reply:
Yep something like either of those. I suppose a slight pref for the latter.